decentraland/helpers/storage.js

```diff
@@ -1,3 +1,5 @@
+// Note: If a data wasn't found, returning null instead of default state of the
```

Comment:
a data -> a piece of data
OR
a data -> data

Reply:
piece of data, just commited

Reply:
Cool, thanks! Resolving this